src/components/FieldMapping/FieldMapping.tsx

```diff
@@ -1,45 +1,63 @@
-import { H2, P2, Stack, useDeskproAppTheme } from "@deskpro/app-sdk";
+import { H1, H2, Stack, useDeskproAppTheme } from "@deskpro/app-sdk";
 import { GreyTitle, StyledLink } from "../../styles";
-import { CustomTag } from "../CustomTag/CustomTag";
 import { HorizontalDivider } from "../HorizontalDivider/HorizontalDivider";
 import { LogoAndLinkButton } from "../LogoAndLinkButton/LogoAndLinkButton";
-import { TwoColumn } from "../TwoColumns/TwoColumns";
+import { SideColumns } from "../SideColumns/SideColumns";
+import lineItemJson from "../../mapping/lineitem.json";
+import { IJson } from "../../types/json";
+import { ReactElement } from "react";
+import { mapFieldValues } from "../../utils/mapFieldValues";
+
+const SpaceBetweenFields = ({
+  field: field,
+}: {
+  field: {
+    key: string | number;
+    value: string | number | ReactElement;
+  };
+}) => {
+  return (
+    <Stack
+      style={{
+        justifyContent: "space-between",
+        width: "100%",
+      }}
+    >
+      <H1 style={{ fontSize: "14px" }}>{field.key}:</H1>
```

Comment:
Please, don't use hardcoded value

Reply:
what do you suggest, making a var with string 14px?

Reply:
Use typographic (P1, P2, ...) components.